flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -442,6 +455,16 @@ private String operatorName(String suffix) {
           throw new RuntimeException("Unrecognized write.distribution-mode: " + writeMode);
       }
     }
+
+    private void buildProperties() {
```

Comment:
I think this should accept a Table instance and return a merged properties map.

Reply:
done